docs/bot/systemd.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 4
+sidebar_label: Installation with Docker and systemd
+---
+
+# Installation with Docker and systemd
+
+If docker is installed on your system, installing Draupnir is as
+simple as creating the following systemd unit file.
+
+:::warning[TODO]
+
+Questions:
+
+1. Do we need a specific user to write to the data directory? Does
+docker even work well with that?
+
+:::
```

Comment:
This is unnecessary, docker runs everything as root sure but the directories that the container has access to is constrained to the volumes that are mounted